src/interceptors/add_multitenant_auth_header.js

```diff
@@ -0,0 +1,19 @@
+/**
+   Read `clientSecret` from `ctx`. Then construct Authorize header and add it to `headers`.
+
+   Changes to `ctx`:
+
+   - Add `headers.Authorize`
+ */
+export default class AddMultitenantAuthHeader {
+  enter(ctx) {
+    const { clientSecret, headers = {} } = ctx;
+
+    if (!clientSecret) {
+      throw new Error('clientSecret is missing from the context');
+    }
+
+    const authHeaders = { Authorization: `Bearer ${clientSecret}` };
+    return { ...ctx, headers: { ...headers, ...authHeaders } };
+  }
+}
```

Comment:
Comment: New line missing. Editors usually add that automatically. Maybe you have some setting that isn't turned on?